spec/models/protocol_spec.rb

```diff
@@ -177,4 +177,255 @@
       expect{ study.push_to_epic(EPIC_INTERFACE) }.to change(EpicQueueRecord, :count).by(1)
     end
   end
+
+  describe "#with_status" do
```

Comment:
Recommendation: Break these out into individual specs. Testing 1 method/scope per spec file is easier to read. We do this for some models and most controllers.